code/game/response_team.dm

```diff
@@ -112,6 +76,8 @@ var/ert_request_answered = 0
 
 	// Respawnable players get first dibs
 	for(var/mob/dead/observer/M in ert_candidates)
+		if(jobban_isbanned(M, ROLE_TRAITOR) || jobban_isbanned(M, "Security Officer") || jobban_isbanned(M, "Captain") || jobban_isbanned(M, "Cyborg"))
```

Comment:
More of a policy thing but shouldn't being silicon job-banned just prevent you from being the silicon role? Seems a bit heavy handed to stop them being ERT entirely. Especially if the admin doesn't choose a cyborg anyway.

Reply:
ERT is an all-access role with considerable firepower that operates outside the normal chain of command. I got a request to make an Eng job ban block Eng ERT, a sec job ban block sec ERT, etc... but that was too complicated. Also, I am of the opinion that ANY SORT of job ban should prevent you from playing ERT. So, this is a stopgap measure to at least ensure the most problematic types of bans are screened out.

Reply:
Fair enough 👍